kafka/4.0/modules/ROOT/pages/kafka-connector-reference.adoc

```diff
@@ -606,7 +606,8 @@ This source supports the consumption of messages from a Kafka cluster, producing
 === For Configurations
 * <<consumer-config>>
 
-
+[NOTE]
+The *Batch Message Listener* source does not support configurable streaming strategies, as the source takes a batch of records as an input stream. The streaming strategy configuration is non-repeatable-stream by default for the source.
```

Comment:
```suggestion
The *Batch Message Listener* source does not support configurable streaming strategies because the source takes a batch of records as an input stream. The streaming strategy configuration is non-repeatable-stream by default for the source.
```